### PR TITLE
Hello Microverse Testing featues:-

### PR DESCRIPTION
**Pull request by misikira**
- HTML and css file
- liters for fixing styling issues and hintrc for html
- readme file to tell you the full picture of this project
- files ignored by git
- other configuration files and info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```
Comment (ABDULALI3468) [inline README.md:33]: Please consider removing the following line from the README to make it professional. Moreover, if you want to give credits to microverse the `Acknowledgements` section is for this purpose.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+I would like to thank
+I would like to thank code reviwers and coding partner
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+Copyright (c) [2023] [misikir teka]
+MIT License
+find more here(<a href="https://choosealicense.com/licenses/mit/">find more here</a>)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ABDULALI3468) [inline README.md:258]: Please consider adding a proper heading of `LICENSE`. Then add a LICENSE file to your Repository and attach it to the link just as done in the [Template](https://github.com/microverseinc/readme-template)

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @misikira , 



Indeed you have done an exceptional job but still, there are some issues that need to be addressed to complete this project 🥇 


## Highlights 
✅All linters are checked.
✅Project is professionally documented.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

**1-** Please consider always using the name of your feature branch as a way to highlight what you did it in. In this case, your branch name is `Landing-Page`  though it's a good name but in your project, the landing page has nothing to do with the project.

> Consider checking out from your current branch to a new branch and for this case, you can name it `hello-microverse` or something else that describes what you did in the branch. Remember in the new PR which you will open for another review add the link of this PR in that PR's Description.

**2-** Please consider adding a professional title to your pull request(PR) to make it professional. It can be the following:-

```
Milestone1:- Hello Microverse Project
```

**3-** It's a project requirement to have `test.md` file in the `.gitginore file`. Though you added it but still it can be found in our current branch. Consider reviewing that.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @misikira , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**